kobo/settings/base.py

```diff
@@ -310,9 +310,23 @@
         30,
         "Number of days to keep asset snapshots"
     ),
+    'FREE_TIER_THRESHOLDS': (
+        json.dumps({
+            'storage': int(5 * 1024 * 1024 * 1024 * 0.9),  # 90 % of 5 GB
```

Comment:
Why 90%?

Reply:
Totally random.

Reply:
Let's use a better default. Either our current internal plan or what we think is a better default for self host users looking to cut down on single users taking up system resources. I don't think 90% makes sense. We could later choose to notify users at 90% usage, but I read this setting as the actual limit at which you cannot submit more (not yet implemented).

Reply:
Agreed. I've changed them to examples in Notion.